pandas/tests/io/parser/test_parse_dates.py

```diff
@@ -1290,17 +1290,17 @@ def test_parse_dates_empty_string(all_parsers):
     tm.assert_frame_equal(result, expected)
 
 
+@pytest.mark.parametrize(
+    'reader',
```

Comment:
You'll need to pass `reader` as an argument to the function, check the pytest docs on parametrisation

Reply:
> Thanks for working on this!
> 
> I've just left some comments - did you manage to run the test locally?

I didn't think to do that since I am new to testing, but I have now. 

Thanks for your help!